tests/console/python310_version_check.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: python310
+# Summary: Run python310 testsuite
+# - Test suitable only for SLE15SP4+
+# - Check that Python 3.6 is the main version installed
+# - Check that Python 3.10 is available to install
+# - Run some basic test (man_or_boy.py)
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use version_utils;
+use utils "zypper_call";
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    if (is_sle('>=15-SP4')) {
+        my $python_ver = script_output("rpm -q python3 | awk -F \'-\' \'{print \$2}\' | awk -F \'.\' \'{print \$1\".\"\$2}\'");
+        if ($python_ver ne "3.6") {
+            die("Python default version " . $python_ver . " differs from 3.6");
+        }
+        my $arch = get_var('ARCH');
+        record_info("Python3 module", "Activate SLE Python3 module");
+        assert_script_run("SUSEConnect -p sle-module-python3/15.4/$arch");
+    }
+    elsif (is_tumbleweed) {
+        my $python_ver = script_output("rpm -q python38 | awk -F \'-\' \'{print \$2}\' | awk -F \'.\' \'{print \$1\".\"\$2}\'");
+        if ($python_ver ne "3.8") {
```

Comment:
same question here, why do you check the current version and fail the test if it is different?